app/jobs/regular/toggle_topic_closed.rb

```diff
@@ -5,6 +5,8 @@ class ToggleTopicClosed < ::Jobs::Base
     def execute(args)
       topic_timer = TopicTimer.find_by(id: args[:topic_timer_id] || args[:topic_status_update_id])
       state = !!args[:state]
+      opts = { silent: args[:silent] }
```

Comment:
A minor thing: if `opts` is used only in a single place (`topic.update_status` further down) I'd just roll this hash into that method call.